### PR TITLE
Add TIXEL manipulator motors

### PR DESCRIPTION
**Pull request by jozamudi**
## Description

Added TIXEL Manipulator motors to LAMPMagneticBottle

## Motivation and Context

https://jira.slac.stanford.edu/browse/ECS-1143

## How Has This Been Tested?

I opened the tmo_lamp using typhos. I moved the motors using the GUI.

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: pcdsdevices/lamp_motion.py [hunk 1]
```diff
@@ -73,6 +73,11 @@ class LAMPMagneticBottle(BaseInterface, GroupDevice):
     magnet_y = Cpt(BeckhoffAxis, ':MMS:06', kind='normal')
     magnet_z = Cpt(BeckhoffAxis, ':MMS:04', kind='normal')
 
+    # TIXEL Manipulator Motors
+    tixel_x = Cpt(BeckhoffAxis, ':MMS:14', kind='normal')
```
Comment (klauer) [inline pcdsdevices/lamp_motion.py:77]:
```suggestion
    tixel_x = Cpt(BeckhoffAxis, ':MMS:14', kind='normal', doc='Tixel manipulator x-axis')
```

File: docs/source/upcoming_release_notes/1154-DOC:_add_TIXEL_Manipulaotr_Motors_to_Magenetic_Bottle.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- N/A
```
Comment (klauer) [inline docs/source/upcoming_release_notes/1154-DOC:_add_TIXEL_Manipulaotr_Motors_to_Magenetic_Bottle.rst:30]:
```suggestion
- jozamudi
```

File: docs/source/upcoming_release_notes/1154-DOC:_add_TIXEL_Manipulaotr_Motors_to_Magenetic_Bottle.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+Added TIXEL Manipulator motors to LAMPMagneticBottle
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1154-DOC:_add_TIXEL_Manipulaotr_Motors_to_Magenetic_Bottle.rst:14]:
```suggestion
- Added TIXEL Manipulator motors to LAMPMagneticBottle
```
Needs the bullet point to work in our release note gathering script

File: pcdsdevices/lamp_motion.py [hunk 4]
```diff
@@ -73,6 +73,12 @@ class LAMPMagneticBottle(BaseInterface, GroupDevice):
     magnet_y = Cpt(BeckhoffAxis, ':MMS:06', kind='normal')
     magnet_z = Cpt(BeckhoffAxis, ':MMS:04', kind='normal')
 
+    # TIXEL Manipulator Motors
+    tixel_x = Cpt(BeckhoffAxis, ':MMS:14', kind='normal',
+                  doc='Tixel manipulator x-axis')
```
Comment (klauer) [inline pcdsdevices/lamp_motion.py:78]: 👍 for adding the `doc` to `tixel_x` - but would you mind doing it for the others as well? The suggestion was a kind of hint

File: docs/source/upcoming_release_notes/1154-add_TIXEL_Manipulaotr_Motors_to_Magenetic_Bottle.rst [hunk 5]
```diff
@@ -0,0 +1,30 @@
+1154 DOC: add TIXEL Manipulaotr Motors to Magenetic Bottle
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/1154-add_TIXEL_Manipulaotr_Motors_to_Magenetic_Bottle.rst:1]: minor funny typos here but I don't think this ends up in the final release docs so it's ok

**Review by ZLLentz — APPROVED** [review]
Looks good to me after the doc updates!

**Review by klauer — APPROVED** [review]
Thanks for doing the fix-ups 👍

**Pull request merged**